tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.scss

```diff
@@ -51,6 +51,10 @@ p {
   display: flex;
   justify-content: space-between;
   padding: 2px 12px;
+
+  @include tb-dark-theme {
+    background-color: #f5f6f722;
```

Comment:
This isn't supposed to be a transparent element.  If we want to make it clear from reading that it's the same hue/saturation with higher brightness, consider hsl()?